src/index.tsx

```diff
@@ -92,11 +81,16 @@ const mapDispatchToProps = (dispatch: Dispatch<XCSpyState>) => {
   };
 };
 
-const App = connect(mapStateToProps, mapDispatchToProps)(XCSpyApp);
+XCSpyApp = connect(mapStateToProps, mapDispatchToProps)(XCSpyApp);
 
 ReactDOM.render(
   <Provider store={store} >
-    <App />
+    <Router>
+      <div>
+        <Route exact path="/" component={XCSpyApp} />
```

Comment:
Best practice: you should try to define the routes in a separate files (along with path constants).
Have a look at the command center project